acra/security-controls/encryption/_index.md

```diff
@@ -2,3 +2,129 @@
+This page describes how to configure it and what you will get as a result.
+
+## Behavior
+
+AcraServer works as a proxy between database client and the database itself.
+When a column is configured to be encrypted, AcraServer will transparently encrypt/decrypt its value.
+However, there are some caveats:
+* In simplest scenario, AcraServer clients won't see any difference: they `INSERT` plaintext,
+  they get plaintext in `SELECT` queries, just like with direct connection to database.
+  At the same time, the data in the database itself will be stored in encrypted form.
+  And if the attacker has full access to the database (but not to the AcraServer),
+  he won't be able to magically bypass the encryption, no keys means no data.
+* Every AcraServer client has own unique identifier (usually derived during TLS or Secure Session handshake),
+  and a keystore associated with that identifier.
+  If the keystore does not contain keys that were used to encrypt requested data,
+  the data will be returned "as is", just like it is stored in database in encrypted form.
+* Apart from client IDs, encryption keys can be bound to [zones]({{< ref "/acra/security-controls/zones/_index.md" >}}),
+  in this case clients should set proper zone ID in requests or AcraServer will be unable
+  to decrypt data because of the same reason as with client ID, no proper keys.
+  Alternatively, zone ID can be stored in database as a separate column.
+* When [masking]({{< ref "/acra/security-controls/masking/_index.md" >}}) is enabled,
+  part of data won't be encrypted (depending on configuration) and
+  the response will be different depending on whether the client has proper keys as well.
+* When [tokenization]({{< ref "/acra/security-controls/tokenisation/_index.md" >}}) is enabled,
+  returned values will be anonymized, they will be different from the actual plaintext.
+
+You don't have to run a single AcraServer instance and give it all the keys.
+Many instances can be launched, each responsible for different tables and/or clients and/or zones.
+
+For example, you can have few clusters, each responsible for
+protection of different data based on its confidentiality level:
+* Few Acra servers that process common requests like "find user by email" or "get his/her name"
+* Dedicated Acra servers for specific geographic regions, with different keys used to
+  encrypt user activity based on his/her location at that monent
+
+When combined with proper network settings, you can end up with a complex system
+where each component can only access data it was created to work with.
+And when combined with other features like
+[SQL firewall]({{< ref "/acra/security-controls/sql-firewall/_index.md" >}}) and/or
+[audit logging](/acra/audit-log-INVALID),
+it will be impossible for somebody to perform malicuous activity without being noticed.
+
+## Configuration
+
+In order to work properly, AcraServer needs to know the schema of the database.
```

Comment:
actually, AcraServer requires schema of the database for transparent encryption, masking and tokenization. For transparent decryption it doesn't require because it is default behavior. And may be used AcraWriter + AcraServer without any config file